python/cudf/cudf/pylibcudf_tests/test_string_find.py

```diff
@@ -0,0 +1,177 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import numpy as np
+import pyarrow as pa
+import pytest
+from utils import assert_column_eq
+
+import cudf._lib.pylibcudf as plc
+from cudf._lib.scalar import DeviceScalar
+
+
+@pytest.fixture()
+def plc_col():
+    data = pa.array(
+        ["AbC", "de", "FGHI", "j", "kLm", "nOPq", None, "RsT", None, "uVw"]
+    )
+    return plc.interop.from_arrow(data)
+
+
+@pytest.fixture
+def find_target_column():
+    return plc.interop.from_arrow(
+        pa.array(["A", "d", "F", "j", "k", "n", None, "R", None, "u"])
+    )
+
+
+@pytest.fixture
+def contains_target_column():
+    return plc.interop.from_arrow(
+        pa.array(["a", "d", "F", "j", "m", "q", None, "R", None, "w"])
+    )
+
+
+@pytest.fixture
+def starts_with_target_column():
+    return plc.interop.from_arrow(
+        pa.array(["A", "d", "F", "j", "k", "n", None, "R", None, "u"])
+    )
+
+
+@pytest.fixture
+def ends_with_target_column():
+    return plc.interop.from_arrow(
+        pa.array(["C", "e", "I", "j", "m", "q", None, "T", None, "w"])
+    )
```

Comment:
Do these actually need to be different? If it's easy to rewrite the inputs to the tests to use the same fixture, we might as well. It makes patterns easier to spot when there's only a single piece of data used for all the testing.

Reply:
Concatenated these into a mega fixture